hangman/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
you have a package [target](https://doc.rust-lang.org/cargo/reference/cargo-targets.html) in your cargo toml, but I might make it a "lib" and a "bin" so someone else could build a separate UI on top of your GameState, but the same repository could be used to distribute the playable terminal-based binary.